packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx

```diff
@@ -46,6 +47,19 @@ export const YppDashboardMainTab: FC<YppDashboardMainTabProps> = ({ currentTier
           }
         />
       )}
+      <StyledBanner
+        title="When rewards are paid out?"
+        icon={<SvgAlertsInformative24 />}
+        description={
+          <>
+            Once your channel is verified, you will qualify for the rewards. Payouts are made on a weekly basis, every
```

Comment:
let's change the text to "Payouts are made on a weekly basis, every Friday, for the previous calendar week. Your first payment will involve the reward for the sign up of XXXX JOY after your channel is verified by the program operators."